testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -496,6 +496,12 @@ testModule('option', moduleConfig, () => {
 
             assert.strictEqual(this.$wrapper.attr('someAttr'), 'someValue');
         });
+
+        test('does not override old clases', function(assert) {
+            this.overlay.option('wrapperAttr', { class: 'newClass' });
+
+            assert.ok(this.$wrapper.hasClass(OVERLAY_WRAPPER_CLASS));
+        });
```

Comment:
let's test that classes do not stack.
-> `wrapperAttr: { class: '1' }`
-> `wrapperAttr: { class: '2' }`
Result classes: only `2`, not `1 2`

Reply:
done